scipy/stats/_odds_ratio.py

```diff
@@ -60,23 +60,25 @@ def _solve(func):
     return nc
 
 
-def _nc_hypergeom_mean_inverse(x, total, ngood, nsample):
+def _nc_hypergeom_mean_inverse(x, M, n, N):
     """
-    For the given total, ngood, and nsample, find the noncentrality
-    parameter of Fisher's noncentral hypergeometric distribution whose
-    mean is x.
+    For the given noncentral hypergeometric parameters x, M, n,and N
+    (table[0,0], total, row 0 sum and column 0 sum, resp., of a 2x2
+    contingency table), find the noncentrality parameter of Fisher's
+    noncentral hypergeometric distribution whose mean is x.
     """
-    nc = _solve(lambda nc: nchypergeom_fisher.mean(total, ngood, nsample,
-                                                   nc) - x)
+    nc = _solve(lambda nc: nchypergeom_fisher.mean(M, n, N, nc) - x)
     return nc
 
 
 def _hypergeom_params_from_table(table):
+    # The notation M, n and N is consistent with stats.hypergeom and
+    # stats.nchypergeom_fisher.
```

Comment:
You were right, this is exactly not what I would have done. But I compared these against the reference, and this parameterization seems equivalent.